### PR TITLE
validator fix

### PR DESCRIPTION
**Pull request by lmahanand**
Required to change the way validator node receives child blocks from supervisor node since supervisor node has few issues related to data not getting persisted in blockchain and account states.

### CODE REVIEW

File: cmd/validator/messageplugin.go [hunk 1]
```diff
@@ -14,8 +14,8 @@ import (
 type HerdiusMessagePlugin struct{ *network.Plugin }
 
 func (state *HerdiusMessagePlugin) Receive(ctx *network.PluginContext) error {
-	contex := network.WithSignMessage(context.Background(), true)
-
+	//contex := network.WithSignMessage(context.Background(), true)
+	fmt.Println("msg has arrived: ", ctx.Message())
```
Comment (cuonglm) [inline cmd/validator/messageplugin.go]: Use log instead of fmt

Comment (lmahanand) [reply cmd/validator/messageplugin.go]: Changing the message. It is not actually required to change the print the complete message.

File: cmd/validator/messageplugin.go [hunk 2]
```diff
@@ -58,10 +55,14 @@ func (state *HerdiusMessagePlugin) Receive(ctx *network.PluginContext) error {
 		// Sign and vote the child block
 		err = vService.Vote(ctx.Network(), ctx.Network().Address, mcb)
 		if err != nil {
-			ctx.Network().Broadcast(contex, &blockProtobuf.ConnectionMessage{Message: "Failed to get vote"})
+			if voteErr := ctx.Reply(network.WithSignMessage(context.Background(), true), &blockProtobuf.ConnectionMessage{Message: "Failed to get vote"}); voteErr != nil {
+				return fmt.Errorf(fmt.Sprintf("Failed to reply to client :%v", voteErr))
```
Comment (cuonglm) [inline cmd/validator/messageplugin.go]: just remove fmt.Sprint, fmt.Errorf is enough

File: cmd/validator/messageplugin.go [hunk 3]
```diff
@@ -58,10 +55,14 @@ func (state *HerdiusMessagePlugin) Receive(ctx *network.PluginContext) error {
...
+			}
 		}
 
-		ctx.Network().Broadcast(contex, mcb)
+		if err := ctx.Reply(network.WithSignMessage(context.Background(), true), mcb); err != nil {
+			return fmt.Errorf(fmt.Sprintf("Failed to reply to client :%v", err))
```
Comment (cuonglm) [inline cmd/validator/messageplugin.go]: Same as above

**Pull request merged**